Description: Remove dev-dependency to cargo-husky
Forwarded: not-needed
Author: Antonin Delpeuch <antonin@delpeuch.eu>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -26,10 +26,6 @@ repository = "https://github.com/nikolay
 [dependencies.rand]
 version = "0.8"
 features = ["small_rng"]
-[dev-dependencies.cargo-husky]
-version = "1"
-features = ["prepush-hook", "precommit-hook", "postmerge-hook", "run-cargo-test", "run-cargo-check", "run-cargo-clippy", "run-cargo-fmt"]
-default-features = false
 
 [dev-dependencies.doc-comment]
 version = "0.3"
